test/anvil/anvil-global-setup.ts

```diff
@@ -66,7 +66,7 @@ export const ANVIL_NETWORKS: Record<NetworksWithFork, NetworkSetup> = {
         rpcEnv: 'SEPOLIA_RPC_URL',
         fallBackRpc: 'https://sepolia.gateway.tenderly.co',
         port: ANVIL_PORTS.SEPOLIA,
-        forkBlockNumber: 7164489n,
+        forkBlockNumber: 7167280n, // Matt moved this blockNo after deploying new partial boosted pool
```

Comment:
Had to move this block number to get the `addLiquidityPartialBoosted.integration.test.ts` test to pass, but as you can see it causes `priceImpact.V3.integration.test.ts` to fail. Maybe there was a trade in the pool?

 I'm not sure if it is okay to move the hardcoded values to fix
 
![image](https://github.com/user-attachments/assets/af8399bc-2671-49e7-93db-04b96e002830)